lib/pbench/test/functional/server/test_connect.py

```diff
@@ -13,6 +16,7 @@ def test_connect(self, server_client: PbenchServerClient):
         endpoints = server_client.endpoints
         assert endpoints
         assert "api" in endpoints
+        assert "authentication" in endpoints
```

Comment:
In lines 23-32 we go on to test `"api"` and `"uri"` much more specifically.  Do we have additional checks that we can assert on `"authentication"`?  (E.g., we're going to need `"issuer"`, `"client"`, `"realm"`, and `"secret"`, right?  And, is there anything that we can assert about their respective values?)

Reply:
The `secret` is optional but yeah I should add assertions for all other fields.